Frontend/ui-customer/src/Helpers/Utils.js

```diff
@@ -513,4 +513,8 @@ export const getMockFollowerName = function() {
     currentMockName = msg.split(" ")[0];
   }
   return currentMockName;
+}
+
+export const getAppName = function(str) {
```

Comment:
wasn't it already present? and if not also, as this is from window we already have a function to fetch values from window variable please use that